webpack.config.js

```diff
@@ -235,11 +239,6 @@ if ( calypsoEnv === 'development' ) {
 	}
 } else {
 	webpackConfig.entry.build = path.join( __dirname, 'client', 'boot', 'app' );
-	webpackConfig.plugins = webpackConfig.plugins.concat( [
-		new webpack.HashedModuleIdsPlugin(),
-		new WebpackChunkHash(),
```

Comment:
unsure if we need chunkhash if we also have named modules and hashed chunks. doesn't appear so, but testing is needed.